reddit_liveupdate/templates/liveupdatehappeningnowbar.html

```diff
@@ -0,0 +1,22 @@
+<%!
+    from r2.lib.template_helpers import format_number
+%>
+
+<a href="/live/${thing.event._id}" class="liveupdate-happening-now">
+  <div class="rank-spacer"></div>
+  <div class="midcol-spacer"></div>
+  <span class="happening-now-logo"></span>
+
+  <div id="liveupdate-statusbar" class="status-bar live happening-now">
+    <p class="state">${_('LIVE')}</p>
+    <p role="presentation">&nbsp;-&nbsp;</p>
+    <p class="description">${thing.event.title}</p>
+    <p class="watching">
+    <%
+      num_watching = thing.event.active_visitors - (thing.event.active_visitors % 100)
```

Comment:
It's primarily so that folks don't think it will live update like reddit live does, but for caching purposes, too.

I knew about the boolean, but the tilde's purpose here is a little different: To specifically make folks feel like this isn't a number that will be updating.